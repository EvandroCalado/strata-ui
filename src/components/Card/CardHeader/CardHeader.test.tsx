import { render } from '@testing-library/react';
import { CardHeader } from '.';

describe('<CardHeader />', () => {
  it('should render a snapshot', () => {
    const { container } = render(<CardHeader />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
