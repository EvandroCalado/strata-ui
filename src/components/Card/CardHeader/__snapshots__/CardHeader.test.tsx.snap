// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<CardHeader /> > should render a snapshot 1`] = `
<header
  class=""
/>
`;
